### ISSUE TITLE
When creating a job, in the job form, strip out @ from the beginning of github & twitter usernames

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/opensourcedesign.github.io#119]
Those things break frontmatter.

**Comment by jdittrich** [comment]
can we mask/escape it?

**Comment by simonv3** [comment]
@jdittrich how do you mean?

**Comment by jdittrich** [comment]
In HTML, one can replace @ with  "\&#64;" – which is displayed as @ (naturally, one needs to know which @ we can replace and which need to be interpreted in the frontmatter)

**Comment by simonv3** [comment]
I don't think that would work, because if they include it in the form input field and we don't strip it, and we point it to their GitHub profile ourselves, then their profile might be github.com/&#64simonv3, which would be incorrect. I think it's better to just remove it all together.

**Comment by Erioldoesdesign** [comment]
I just got this errorbug trying to submit a job so I screen capped it:

<img width="1159" alt="Screenshot 2020-06-15 at 14 12 21" src="https://user-images.githubusercontent.com/11681324/84661749-b1792480-af12-11ea-9a70-9863a065548f.png">

It happened if I put into the github username field both:

@myusername
myusername

the '@' symbol made no difference :/ both errors

**Comment by jdittrich** [comment]
Closed upon "Close decision by end of 2025"-Tag
